lib/publishing/nuget/publish.sh

```diff
@@ -80,11 +80,17 @@ for NUGET_PACKAGE_PATH in $(find dotnet -name *.nupkg -not -iname *.symbols.nupk
 
         if [ -f "${NUGET_PACKAGE_BASE}.symbols.nupkg" ]; then
             # Legacy mode - there's a .symbols.nupkg file that can't go to the NuGet symbols server
-            $dotnet nuget push $NUGET_PACKAGE_NAME -k $NUGET_API_KEY -s $NUGET_SOURCE -ss $NUGET_SYMBOL_SOURCE | tee ${log}
+            dotnet nuget push $NUGET_PACKAGE_NAME -k $NUGET_API_KEY -s $NUGET_SOURCE -ss $NUGET_SYMBOL_SOURCE --force-english-output --skip-duplicate | tee ${log}
         else
             [ -f "${NUGET_PACKAGE_BASE}.snupkg" ] || echo "⚠️ No symbols package was found!"
             # The .snupkg will be published at the same time as the .nupkg if both are in the current folder (which is the case)
-            $dotnet nuget push $NUGET_PACKAGE_NAME -k $NUGET_API_KEY -s $NUGET_SOURCE | tee ${log}
+            dotnet nuget push $NUGET_PACKAGE_NAME -k $NUGET_API_KEY -s $NUGET_SOURCE --force-english-output --skip-duplicate | tee ${log}
+        fi
+        
+        # we need to check PIPESTATUS since "tee" is the last command which is always likely to succeed
+        if [ ${PIPESTATUS[0]} -ne 0 ]; then
+            echo "❌ Error publishing ${NUGET_PACKAGE_NAME} to NuGet"
+            exit 1
         fi
```

Comment:
Since this now runs with `pipefail`, we need not check this anymore - the option's purpose is to make the whole pipe sequence result in an error if any of its stages is not exiting with status 0!

Reply:
ahh i get it now